release-team/role-handbooks/release-notes/editing-flow.md

```diff
@@ -0,0 +1,245 @@
+will be created in the release directory after saving the file and exiting the editor.
+
+__Note:__ A valid map file should have at least the PR number and the releasenote struct.
+
+If the user makes an error (a YAML syntax error for example), krel will prompt the user
+to retry editing the note:
+
+```
+ERRO The yaml code does not have a PR number      
+- An error occurred while editing PR #92546. Try again? (y/n)
+```
+
+#### Exiting the review loop
+
+When in the interactive cycle, krel will present the user with every release note that
+has not yet been reviewed. It is not necessary to review all the notes in one go, at
+any time the user may hit Ctrl+C and the interactive flow will exit at that point.
+
+To continue the editing process at a later time, `krel` will save its state
+in the release directory. It will store information about each review session inside of
+the `release-notes/sessions/`. The JSON files include the user's data and the release
+notes reviewed during the session.
+
+At this point, krel will create the current release notes Draft. This file is the 
+latest version of the automatically generated release notes and is made public for
+any interested party to review. 
+
+### Submitting changes back
+
+After exiting the interactive mode, krel will prompt the user to create a pull request
+submitting all changes back to the sig-release repository:
+
+```
+INFO Release Notes Draft written to /tmp/k8s-730127624/releases/release-1.19/release-notes-draft.md 
+Create pull request with your changes? (y/n): 
+```
+
+If the user choses so, krel will create the pull request on their behalf by creating a
+branch in their sig-release fork, pushing the changes and creating the PR in GitHub. 
+
+Note that the PR will include the release notes draft with all changes defined in
+the maps during the editing flow. All the mapping files and session data will be
+submitted in the same PR.
+
+If the user cancels the PR, the local fork of sig-release will be left in a temporary
+directory with all changes done by krel. To submit the changes back to sig-release, 
+the user will be required to push and create the PR manually. To assist them with this
+process, krel will show some instructions:
+
+```
+Create pull request with your changes? (y/n): n
+Pull request creation was canceled. Your local copy of k/sig-release
+has not been deleted for you to review your changes. You may
+push your changes to your fork and create the PR from there.
+
+Your fork of kubernetes/sig-release was cloned here:
+/tmp/k8s-730127624
+
+WARN Pull request canceled. Local changes were not pushed back.
+```
+
+Needless to say, all editing not PR'ed back to sig-release will be lost.
+
+### Re-running the workflow
+
+It is highly recommended that the Release Notes team runs the editing flow
+frequently (say on a weekly basis). This ensures that the current user has
+a reasonable amount of work to do.
```

Comment:
> This ensures that the current user has a reasonable amount of work to do.

I don't think I understand what this means. A reasonable amount of work to do?

Reply:
I think the sentiment meant here is that Release Notes shadows often don't have much work to do until the tail end of the release cycle and that running the editing flow weekly allows Release Notes shadows to be more involved. My thought here is that running the editing flow weekly and alternating team members to run it is good to get team members familiar with the editing flow and with incoming PRs instead of only digging into the PRs towards the end of the cycle.

Reply:
OK, I've tried to express these. ptal